api/src/main/java/org/coner/core/api/entity/CompetitionGroupApiEntity.java

```diff
@@ -66,27 +67,16 @@ public void setResultTimeType(String resultTimeType) {
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
-
         CompetitionGroupApiEntity that = (CompetitionGroupApiEntity) o;
-
-        if (grouping != null ? !grouping.equals(that.grouping) : that.grouping != null) return false;
-        if (handicapFactor != null ? !handicapFactor.equals(that.handicapFactor) : that.handicapFactor != null)
-            return false;
-        if (id != null ? !id.equals(that.id) : that.id != null) return false;
-        if (name != null ? !name.equals(that.name) : that.name != null) return false;
-        if (resultTimeType != null ? !resultTimeType.equals(that.resultTimeType) : that.resultTimeType != null)
-            return false;
-
-        return true;
+        return Objects.equals(id, that.id) &&
```

Comment:
I plugged in some of these, and it's posing some challenges due to some inconsistent use of scale in the `BigDecimal` instances in our tests. See https://issues.apache.org/jira/browse/LANG-393

I think that's exposing an issue we didn't know we had, and I would like to fix it, but I think it's out of scope to this PR.